gcp/api/server.py

```diff
@@ -935,17 +971,22 @@ def _query_by_semver(context: QueryContext, query: ndb.Query, package_name: str,
                      ecosystem: str, purl: PackageURL | None, version: str):
   """Query by semver."""
```

Comment:
Please improve this docstring while you're here: https://google.github.io/styleguide/pyguide.html#s3.8.3-functions-and-methods